src/containers/BooksForm.js

```diff
@@ -1,20 +1,82 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { createBook } from '../actions/index';
 
-const BooksForm = () => {
-  const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+class BooksForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      category: '',
+    };
+    this.baseState = this.state;
```

Comment:
**OPTIONAL**: Interesting, so `this.baseState` is referencing `this.state`, not a copy of it... that can be trouble 🤨... is this really needed?